src/components/resource/show.js

```diff
@@ -328,12 +331,13 @@ class ResourceShow extends Component {
                   </div>
                 </KeyValue>
 
-                {(proxyTypes.isLoading || model.isLoading) ? (
+                {(!hasInheritedProxy || !proxyTypes.request.isResolved || model.isLoading) ? (
```

Comment:
same as above here. probably, there is no situation where a resource can have proxy but not a package containing it but we need to double check on that assumption